Resources/Private/Partials/Search/BatchActions.html

```diff
@@ -43,13 +43,4 @@
             additionalAttributes="{disabled:'disabled'}"
     />
 
-
-    <f:if condition="{dpf:security.kitodoRole()} == 'ROLE_RESEARCHER' || {dpf:security.kitodoRole()} == 'ROLE_LIBRARIAN'">
-        <f:then>
-            <a href="#" id="batchButtonBatchExport" class="btn btn-sm btn-secondary disabled">
-                {f:translate(key: 'manger.workspace.batch.buttonExport', default: 'Exportieren')}
```

Comment:
Do we need to remove the translation key as well?

Reply:
Yes, therefore I removed the key.